compiler/rustc_target/src/spec/targets/aarch64_unknown_none.rs

```diff
@@ -21,7 +21,9 @@ pub fn target() -> Target {
             &["--fix-cortex-a53-843419"],
         ),
         features: "+v8a,+strict-align,+neon,+fp-armv8".into(),
-        supported_sanitizers: SanitizerSet::KCFI | SanitizerSet::KERNELADDRESS,
+        supported_sanitizers: SanitizerSet::KCFI
+            | SanitizerSet::KERNELADDRESS
+            | SanitizerSet::SHADOWCALLSTACK,
```

Comment:
Doesn't this make `SHADOWCALLSTACK` always supported regardless of fixed-x18 flag?

Reply:
No we can't assume that. I should just revert this hunk.